support-frontend/assets/pages/paper-subscription-landing/helpers/getPlans.tsx

```diff
@@ -0,0 +1,262 @@
+import type { ReactNode } from 'react';
+import type { Product } from 'components/product/productOption';
+import { BillingPeriod } from 'helpers/productPrice/billingPeriods';
+import type {
+	FulfilmentOptions,
+	PaperFulfilmentOptions,
+} from 'helpers/productPrice/fulfilmentOptions';
+import type { PaperProductOptions } from 'helpers/productPrice/productOptions';
+import { ActivePaperProductTypes } from 'helpers/productPrice/productOptions';
+import type {
+	ProductPrice,
+	ProductPrices,
+} from 'helpers/productPrice/productPrices';
+import {
+	getDiscountVsRetail,
+	getProductPrice,
+	showPrice,
+} from 'helpers/productPrice/productPrices';
+import type { Promotion } from 'helpers/productPrice/promotions';
+import { finalPrice, getAppliedPromo } from 'helpers/productPrice/promotions';
+import type { TrackingProperties } from 'helpers/productPrice/subscriptions';
+import {
+	sendTrackingEventsOnClick,
+	sendTrackingEventsOnView,
+} from 'helpers/productPrice/subscriptions';
+import { paperCheckoutUrl } from 'helpers/urls/routes';
+import getPlanData from '../planData';
+import { getProductLabel, getTitle } from './products';
+
+const getPriceCopyString = (
+	price: ProductPrice,
+	productCopy: ReactNode = null,
+): ReactNode => {
+	const promotion = getAppliedPromo(price.promotions);
+
+	if (promotion?.numberOfDiscountedPeriods) {
+		return (
+			<>
+				per month for {promotion.numberOfDiscountedPeriods} months{productCopy},
+				then {showPrice(price)} after
+			</>
+		);
+	}
+
+	return <>per month{productCopy}</>;
+};
+
+const getOfferText = (price: ProductPrice, promo?: Promotion) => {
+	if (promo && price.savingVsRetail && promo.discount?.amount) {
+		const discount = getDiscountVsRetail(
+			price.price,
+			price.savingVsRetail,
+			promo.discount.amount,
+		);
+		if (discount > 0) {
+			return `Save ${discount}% on retail price`;
+		} else {
+			return '';
+		}
+	}
+
+	if (price.savingVsRetail && price.savingVsRetail > 0) {
+		return `Save ${price.savingVsRetail}% on retail price`;
+	}
+
+	return '';
+};
+
+const getUnavailableOutsideLondon = (
+	fulfilmentOption: FulfilmentOptions,
+	productOption: PaperProductOptions,
+) =>
+	fulfilmentOption === 'HomeDelivery' &&
+	(productOption === 'Saturday' || productOption === 'Sunday');
+
+// ---- Plans ----- //
+const copy: Record<
+	PaperFulfilmentOptions,
+	Record<PaperProductOptions, JSX.Element>
+> = {
+	HomeDelivery: {
+		Everyday: (
+			<>
+				{' '}
+				for <strong>the Guardian</strong> and <strong>the Observer</strong>,
+				delivered
+			</>
+		),
+		Sixday: (
+			<>
```

Comment:
This copy (and observer) are repeated quite a bit. Perhaps declare as a const and repeat the const to shorten?